### DISCUSSION TITLE
What is the End of life and End of service of OpenPop version 2.0.4.369 ?

### DISCUSSION THREAD

**Discussion by Hemant-700** [cake-build/cake#4145] [Q&A]
Hi,
we are using OpenPop **2.0.4.369** Win, services.
we want to know the  **End of life** and **End of service** of version **2.0.4.369**.

**Comment by augustoproiete** [comment]
@Hemant-700 This is the discussion board of the Cake project - https://cakebuild.net - we are not related to OpenPop. Try asking on https://stackoverflow.com
